source/options.tsx

```diff
@@ -173,10 +174,37 @@ async function highlightNewFeatures(): Promise<void> {
 	void browser.storage.local.set({featuresAlreadySeen});
 }
 
+async function getFeaturesDisabledViaHotfix(): Promise<HTMLElement[]> {
+	const hotfixes = await cache.get<string[][]>('hotfixes');
+	if (!hotfixes) {
+		return [];
+	}
+
+	const {version} = browser.runtime.getManifest();
+	const featureNames: Set<string> = new Set(features.map(({id}) => id));
+	const disabledFeatures = [];
+	for (const [feature, unaffectedVersion] of hotfixes) {
+		const relatedIssue = '#1234';
```

Comment:
This this PR incomplete/draft?

Reply:
That was for the demo screenshot, forgot to remove it :sweat: